QUEST.tsv

```diff
@@ -45,251 +45,251 @@ QUEST_20150317_000044	Items to be deleted
-QUEST_20150317_000138	Klaipeda is a town that lives off its mining. {nl}With Medis Diena, we ought to work on planting trees or cutting them. 
+QUEST_20150317_000137	Even with the world as chaotic as it is, we can't stop working. {nl}It's been four years. Isn't it time to make peace with it and start over?
+QUEST_20150317_000138	Klaipeda is a town that lives off its mining. {nl}With Medis Diena, we ought to work on planting trees or cutting them.
 QUEST_20150317_000139	My old man did this thing where he checked around corners for anything nice. {nl}Sometimes, he'd get lucky and find a nice treasure chest. I wonder who leaves them there?
 QUEST_20150317_000140	Retired Hunter
-QUEST_20150317_000141	I remember this battle in a deep canyon, where soldiers fought a huge monster. {nl}It was amazing how the giant thing was turned around and there were some great spoils, too.
+QUEST_20150317_000141	I remember this battle in a deep canyon, where soldiers fought a huge monster. {nl}Seeing the giant thing fall was amazing, but what was more amazing was the spoils.
 QUEST_20150317_000142	Old Potter
-QUEST_20150317_000143	I put prayers on my earthware, one by one. {nl}That way, the pots will one day be filled with the Goddess' blessings.
+QUEST_20150317_000143	I put a prayer on my earthenware, one by one. {nl}That way, the pots will one day be filled with the Goddess' blessings.
 QUEST_20150317_000144	Swordsmen should have Strength, but you can also have Dexterity and Constitution. {nl}You'll need to think about what to start with.
-QUEST_20150317_000145	Wizards work with Intelligence, but shouldn't forget Spirit for more spellcasting. {nl}You'll have to choose between being balanced or being focused.
+QUEST_20150317_000145	Wizards work with Intelligence, but shouldn't forget Spirit for more spell-casting. {nl}You'll have to choose between being balanced or specializing.
 QUEST_20150317_000146	Archers should have Strength, but Dexterity will prevent problems with movement. {nl}You'll need to think about what to start with.
-QUEST_20150317_000147	Clerics work with Intelligence, but might need a lot of Strength or Spirit. {nl}You should really decide what you want first.
+QUEST_20150317_000147	Clerics work with Intelligence, but might need a lot of Strength or Spirit. {nl}You should really decide what you want, first.
 QUEST_20150317_000148	Doppelsoeldner Master
-QUEST_20150317_000149	The things that are worthwhile will always carry a degree of risk. {nl}Naturally, for big rewards, you'll have to put up with danger.
+QUEST_20150317_000149	Worthwhile things always carry a degree of risk. {nl}Naturally, for the big rewards, you'll have to put up with lots of danger.
 QUEST_20150317_000150	Corsair Master
-QUEST_20150317_000151	I won't help anyone in trouble if there isn't anything in it for me.
+QUEST_20150317_000151	If there isn't anything in it for me, I won't lift a finger.
```

Comment:
"I won't lift a finger" -> to do what?  This change lacks the context of "to help someone in trouble"

The original sentence looks fine as is.